.github/workflows/helm-enterprise.yml

```diff
@@ -54,57 +54,90 @@ jobs:
         run: |
           CHARTS=$(jq -nc --argjson json '${{ env.JSON }}' '${{ env.JQ_FILTER }}')
           echo "json=$CHARTS" >> $GITHUB_OUTPUT
+
   helm_integration:
-    runs-on: [self-hosted, linux, X64]
+    runs-on: [self-hosted]
     needs:
       - build_vars
     name: Helm ${{ matrix.name }} ${{ matrix.values }}
     strategy:
       fail-fast: false
-      max-parallel: 3
       matrix:
         include: ${{ fromJSON(needs.build_vars.outputs.ver_json) }}
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
+
       - uses: azure/setup-helm@v3
         with:
           version: "3.12.1"
+
       - name: Setup aws cli
         run: |
           curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
           unzip awscliv2.zip
           sudo ./aws/install
+
       - name: Setup dnsutils
         run: |
           sudo apt update
           sudo apt install -y dnsutils
-      - name: compute version as expected by deployment script
-        id: values_to_version
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Setup cluster
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-kind@v3.1.0
+
+      - name: Create registries auth secret
         run: |
-          VFILE="${{ matrix.values }}"
-          if [ "${VFILE}" == 'values.yaml' ]; then V='latest'
-          else V=${VFILE%_values.yaml}
-          fi
-          SANITIZED_V=$(echo ${{ matrix.name }} | sed -r 's/[[:punct:]]?(.)[[:alnum:]]*/\1/g')-${V//./}
-          SANITIZED_V=${SANITIZED_V,,}
-          echo "Release prefix to $SANITIZED_V"
-          echo "app_version=$V" >> $GITHUB_OUTPUT
-          echo "app_prefix=${SANITIZED_V}" >> $GITHUB_OUTPUT
-      - uses: >-
-          Alfresco/alfresco-build-tools/.github/actions/dbp-charts/verify-helm@v3.1.0
+          kubectl create secret generic regcred \
+          --from-file=.dockerconfigjson=$HOME/.docker/config.json \
+          --type=kubernetes.io/dockerconfigjson
+
+      - name: Helm install
+        run: |
+          helm dep up ./helm/alfresco-content-services
+          helm install acs ./helm/alfresco-content-services \
+            --set global.tracking.sharedsecret="$(openssl rand -hex 24)" \
+            --set global.known_urls=http://localhost \
+            --set global.alfrescoRegistryPullSecrets=regcred \
+            --set 'global.registryPullSecrets[0]=regcred' \
```

Comment:
Do we have a jira to fix that?

Reply:
not really, culprit is https://github.com/Alfresco/alfresco-process-application-deployment/blob/develop/helm/alfresco-process-application/values.yaml#L10